.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,38 @@
+<!-- Thanks for contributing to our project! We appreciate your time and effort.
+Please fill out the information below to expedite the review and merge of your pull request.
```

Comment:
My intention was to gently push for always having a "fixed issue" section :grin:
So, if there isn't one, please create it. This is because this might clarify better IF there is a need for a PR and what the expected content could be.

Of course, if the PR comes from us, we might have an ECOPROJECT link or not, but better give a good example